data/konbucase.metainfo.xml.in

```diff
@@ -52,26 +52,26 @@
   <url type="translate">https://hosted.weblate.org/projects/rosp</url>
 
   <!-- developer_name has deprecated since AppStream 1.0 -->
-  <developer_name translatable="no">Ryo Nakano</developer_name>
+  <developer_name translate="no">Ryo Nakano</developer_name>
   <developer id="com.github.ryonakano">
-    <name translatable="no">Ryo Nakano</name>
+    <name translate="no">Ryo Nakano</name>
   </developer>
 
   <releases>
     <release version="4.1.2" date="2024-03-02" urgency="low">
-      <description translatable="no">
+      <description translate="no">
         <ul>
           <li>Set minimum window size to prevent it from being cut off</li>
           <li>Send toast when text copied</li>
           <li>Flatpak: Update platform version</li>
-          <li>Metainfo: Mark release descriptions as untranslatable</li>
+          <li>Metainfo: Mark release descriptions as untranslate</li>
```

Comment:
Ah…maybe you replaced all occurrence or "translatable" in this file :sweat_smile:

```suggestion
          <li>Metainfo: Mark release descriptions as untranslatable</li>
```